WalletWasabi.Fluent/Helpers/TransactionHelpers.cs

```diff
@@ -57,7 +57,7 @@ public static BuildTransactionResult BuildTransaction(Wallet wallet, BitcoinAddr
 			payments: intent,
 			feeStrategy: FeeStrategy.CreateFromFeeRate(feeRate),
 			allowUnconfirmed: true,
-			allowedInputs: coins.Select(coin => coin.Outpoint),
+			allowedInputs: coins.Where(coin => coin.IsAvailable()).Select(coin => coin.Outpoint),
```

Comment:
This doesn't look good, for many reasons. Any of these methods shouldn't be touched.

Imagine you see on the send first screen that your balance is 1 BTC and you want to move that, and this method leaves out a few coins.
Or the balance is 1 BTC and you want to send a little bit less (so the fee won't be subtracted from the amount, but will be an additional cost), send first screen will pass without error but after that, you will get (probably an insufficient balance) error just because this filtered out some coins.
And who knows in what other cases it is broken.

Sending coins that are participating in CJ should be prevented way earlier.
1. Send first screen should mention `Available balance` instead of `Balance`.
2. Everywhere where the balance is taken into account, only the available balance should be taken into account.
3. In the pocket selections screen, the CJing coins should be left out of the pockets. (That is a good question what to do in the coin control, maybe it is ok not to show them as well)

Reply:
You are right, it's an awful hack. I will try to trace back where can I put my filtering.

Reply:
I tried to filter beforehand, but it's still questionable IMO. 
If you have time, please take another look.